addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/BigRedirectsScanRule.java

```diff
@@ -106,6 +108,46 @@ private int getPredictedResponseSize(int redirectURILength) {
         return predictedResponseSize;
     }
 
+    private AlertBuilder createBaseAlert() {
+        return newAlert()
+                .setRisk(Alert.RISK_LOW)
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setCweId(201)
+                .setWascId(13);
+    }
+
+    private AlertBuilder createBigAlert(String otherinfo) {
+        return createBaseAlert()
+                .setDescription(Constant.messages.getString(MESSAGE_PREFIX + "desc"))
+                .setOtherInfo(otherinfo)
+                .setSolution(getSolution())
+                .setAlertRef(String.valueOf(PLUGIN_ID) + "-1");
+    }
+
+    private AlertBuilder createMultiAlert(String otherinfo) {
+        return createBaseAlert()
+                .setName(Constant.messages.getString(MESSAGE_PREFIX + "multi.name"))
+                .setDescription(Constant.messages.getString(MESSAGE_PREFIX + "multi.desc"))
+                .setOtherInfo(
+                        Constant.messages.getString(MESSAGE_PREFIX + "multi.extrainfo", otherinfo))
+                .setSolution(getSolution())
+                .setAlertRef(String.valueOf(PLUGIN_ID) + "-2");
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                createBigAlert(
+                                Constant.messages.getString(
+                                        MESSAGE_PREFIX + "extrainfo",
+                                        18,
+                                        "http://example.com",
+                                        318,
+                                        319))
```

Comment:
This should not be duplicated, better have them as parameters in `createBigAlert` than pass the whole other info.